sinch/domains/verification/endpoints/start_verification.py

```diff
@@ -32,20 +35,38 @@ def handle_response(self, response: HTTPResponse) -> StartVerificationResponse:
         super().handle_response(response)
         if self.request_data.method == VerificationMethod.SMS.value:
             return StartSMSInitiateVerificationResponse(
-                **response.body
+                id=response.body.get("id"),
+                method=response.body.get("method"),
+                _links=response.body.get("_links"),
+                sms=SMSResponse(
+                    interception_timeout=response.body["sms"].get("interceptionTimeout"),
```

Comment:
We can expect the server to always return this field, but it's noted as optional in the specification. Wouldn't it be safer to check for the existence of the `sms` property before reading sub-properties?

Reply:
I've added some additional logic.